scripts/gh/show-bump.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Yeah often I would do:
```bash
#! /usr/bin/env nix-shell
#! nix-shell -i bash -p gnused html-xml-utils
```
but since this script is usually running on GitHub actions then we don't have `nix-shell` installed, and so we would need to execute it with `bash ./scripts/gh/show-bump.sh` ...